core/src/main/kotlin/com/imashnake/animite/core/ui/Stats.kt

```diff
@@ -0,0 +1,30 @@
+package com.imashnake.animite.core.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+
+@Composable
+fun <T> StatsRow(
```

Comment:
Does this really need to be a generic?

Reply:
Yeah I kinda followed what I have in `MediaSmall`, I made it generic so I can reuse it as I did for `fun <T> MediaSmallRow`.